src/e2e/puppeteer/__tests__/render-thoughts.ts

```diff
@@ -27,6 +43,12 @@ const testSuite = () => {
       const image = await screenshot()
       expect(image).toMatchImageSnapshot()
     })
+
+    it('initial load on light theme', async () => {
```

Comment:
I actually vote for taking all the Light tests out of the `testSuite` that gets run at different font sizes. Theme and font size are orthogonal, so we don't need to test different themes on different font sizes. Doing so will just slow down the tests.

What do you think?

Reply:
I agree, will take it out, thanks!